### DISCUSSION TITLE
Chat Model dropdown

### DISCUSSION THREAD

**Discussion by luk-bro** [nicobrauchtgit/obsidian-Smart2Brain#96] [Ideas]
Hi Smart2Brain team,

Awesome plugin, I have just one request. Please add more flexibility and allow text input into the chat model selection. OpenAI provides various models; it would be great to experiment with all of them as with the just released GTP4-Turbo.

**Comment by Leo310** [comment]
We will add the missing OpenAI models to the dropdown, but we won't allow text input as it worsens the user experience in our opinion.

**Comment by Leo310** [comment]
I know Its been some time, but it will be supported in the next Release ;)
Wont take long anymore!
